### PR TITLE
feat: [#188601241] Enabled License Status Check Tasks for Unmapped Su…

### PR DESCRIPTION
**Pull request by cbrunefearless**
## Description
Enabled License Status Check Tasks for Unmapped Supported License Types

User Story
As Business.NJ.gov user, I want to be able to check my license application status so that I can check my status and see renewals and reinstatements.

Acceptance Criteria
Given I have entered my business name and address
when accessing my Architecture industry roadmap
then the Authorization Architect Firm license task is rendered
and I can check the status of my Architecture Certificate of Authorization license

Given I have entered my business name and address
when accessing my Landscape Architecture industry roadmap
then the Authorization Landscape Architect Firm license task is rendered
and I can check the status of my Landscape Architecture Certificate of Authorization license

Given I have entered my business name and address
when accessing my Telemarketing industry roadmap
then the Telemarketing license task is rendered
and I can check the status of my Telemarketing license

Given I have entered my business name and address
when accessing my Health Club industry roadmap
then the Health Club Registration license task is rendered
and I can check the status of my Health Club Registration license license

Given I have entered my business name and address
when accessing my Entertainment industry roadmap with selling tickets
then the Entertainment industry roadmap license task is rendered
and I can check the status of my Ticket Broker (Ticket Reseller) license
### Ticket

This pull request resolves [#188601241](https://www.pivotaltracker.com/story/show/188601241).

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation on the engineering documentation website
- [x] I have not used any relative imports
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] I have checked for and removed instances of unused config from CMS
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

**Review by adidner — APPROVED** [review]
Approved with 2 minor clarifications. Works from what I can tell in app.

File: content/src/roadmaps/license-tasks/ticket-broker-reseller-registration.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ agencyId: "nj-consumer-affairs"
 agencyAdditionalContext: "Regulated Business Section"
 industryId: "retail"
 licenseCertificationClassification: "undefined"
-licenseName: ""
+licenseName: "Ticket Brokers"
 summaryDescriptionMd: "You need to register as a ticket broker if you resell admission tickets to places of entertainment and charge a premium, in addition to the price plus the taxes printed on the tickets."
 ---
```
Comment (adidner) [inline content/src/roadmaps/license-tasks/ticket-broker-reseller-registration.md:17]: So this one is hidden behind a profile question. That is intentional right? (it isn't called out in AC)

File: content/src/roadmaps/license-tasks/authorization-architect-firm.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (adidner) [inline content/src/roadmaps/license-tasks/authorization-architect-firm.md:1]: So in the AC it says that these should only be added to road map with an entered address and business name. 

I just checked in a completely up and running state (via the cards). But I didn't need to enter an address or business name is this intended?

Comment (cbrunefearless) [reply content/src/roadmaps/license-tasks/authorization-architect-firm.md:1]: Yes that is fine

File: content/src/roadmaps/license-tasks/ticket-broker-reseller-registration.md (hunk 1 above)
Comment (cbrunefearless) [reply content/src/roadmaps/license-tasks/ticket-broker-reseller-registration.md:17]: Yes that is fine..

**Pull request merged**